server/lib/configuration.js

```diff
@@ -105,7 +105,10 @@ var conf = module.exports = convict({
       doc: 'path to where experiments are stored'
     },
     git: {
-      default: 'github:mozilla/fxa-content-experiments#dev',
+      /**
+      THIS IS JUST FOR TESTS FOR NOW
+       */
+      default: 'github:mozilla/fxa-content-experiments#device-list',
```

Comment:
TO BE REMOVED once https://github.com/mozilla/fxa-content-experiments/pull/32 is merged